src/features/Chat/chatPageApi.ts

```diff
@@ -116,20 +117,26 @@ export const chatApi = baseApi.injectEndpoints({
       }),
     }),
     updateStatus: builder.mutation<unknown, PutStatus>({
-      query: ({ userId, buddyId, status }) => ({
+      query: ({ userId, buddyId, nextStatus }) => ({
         url: `users/${userId}/contacts/${buddyId}`,
         method: 'put',
-        body: { status },
+        body: { status: nextStatus },
       }),
-      async onQueryStarted({ status }, { queryFulfilled }) {
+      async onQueryStarted({ nextStatus, originalStatus }, result) {
         try {
-          await queryFulfilled;
-          toast.success(t(statusUpdateSuccessMessages[status]), {
-            id: `${status}-status-success`,
-          });
+          await result.queryFulfilled;
+          if (originalStatus === 'banned' && nextStatus === 'ok') {
+            toast.success(t('chat:notification.unblockingSuccess'), {
```

Comment:
Should we use `ChatStatus` and `statusUpdateSuccessMessages` instead of strings?

Reply:
Toast updated to use `statusUpdateSuccessMessages` 

Not quite sure where we want to use `ChatStatus` ? 😮

Reply:
Looks good now :+1: